vars/cloudFoundryDeploy.groovy

```diff
@@ -412,17 +413,29 @@ private deploy(def cfApiStatement, def cfDeployStatement, def config, Closure po
         usernameVariable: 'username'
     )]) {
 
+        def postDeployCommand = config.postDeployCommand
+
+        if(postDeployCommand) {
+            echo "[INFO][${STEP_NAME}] Will Execute Post Deploy Command: '${postDeployCommand}'."
+        }
+        else {
+            echo "[INFO][${STEP_NAME}] No Post Deploy Command to Execute."
+        }
+
         def cfTraceFile = 'cf.log'
 
         def deployScript = """#!/bin/bash
             set +x
             set -e
             export HOME=${config.dockerWorkspace}
             export CF_TRACE=${cfTraceFile}
+            export CF_STAGING_TIMEOUT=60
+            export CF_STARTUP_TIMEOUT=20
             ${cfApiStatement ?: ''}
             cf login -u \"${username}\" -p '${password}' -a ${config.cloudFoundry.apiEndpoint} -o \"${config.cloudFoundry.org}\" -s \"${config.cloudFoundry.space}\" ${config.loginParameters}
             cf plugins
             ${cfDeployStatement}
+            ${postDeployCommand}
```

Comment:
We should discuss if we want to execute the postDeployCommand in the same Docker container. E.g. another tool could be required, such as npm/nodejs, which is not part of this image. Maybe you would like to configure another Docker image for that.  However, it might be more difficult for a potential go migration.
@OliverNocon  @CCFenner Opinions?

Reply:
For me the use case is not quite clear. I understood that I can define modules that are loaded from docker images and thus there is no need for a `path` which is removed by this PR. But for what is the `postDeployCommand` necessary? Is it really related?